source/schema.d.ts

```diff
@@ -32,12 +33,13 @@ const userMaskSettings: UserMask = {
 	created: 'show',
 	active: 'show',
 	passwordHash: 'hide',
+	attributes: ['mask', 'show']
 }
 ```
 
 @category Object
 */
-export type Schema<ObjectType, ValueType> = ObjectType extends string
+export type Schema<ObjectType, ValueType, Options extends SchemaOptions = {}> = ObjectType extends string
```

Comment:
Because the `@default` of `recurseIntoArrays` is `true` when missing it probably creates the problems, I see only two use cases in the library that has `@default true` when missing.

- `CamelCaseOptions`, this has the generic default to `{preserveConsecutiveUppercase: true}`
- `GetOptions`, this has the generic default to `{}`

Reply:
You can still make it true by default without using the default parameter. Just invert  the check.

Reply:
Fixed